dataflow/src/main/java/org/checkerframework/dataflow/cfg/visualize/CFGVisualizeLauncher.java

```diff
@@ -35,24 +36,28 @@
  * org.checkerframework.dataflow.cfg.playground.ConstantPropagationPlayground} for another way to
  * use it.
  */
-public class CFGVisualizeLauncher {
+public final class CFGVisualizeLauncher {
+
+    /** Class cannot be instantiated. */
+    private CFGVisualizeLauncher() {
+        throw new AssertionError("Class CFGVisualizeLauncher cannot be instantiated.");
```

Comment:
I'd recommend using `IllegalStateException`

Reply:
Could you pls tell me the reason? The explain of IllegalStateException is "Signals that a method has been invoked at an illegal or inappropriate time. In other words, the Java environment or Java application is not in an appropriate state for the requested operation."

Reply:
It seems to me that it's never appropriate to call this constructor, thus it's always an illegal state. And I'm not sure if it makes sense to throw `AssertionError` without showing any conditions (e.g., `assert x != 1`). It seems the book Effective Java is also using `AssertionError`, is there any explanation for doing so?

Reply:
https://github.com/eisop/checker-framework/blob/master/javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java#L54, I just followed this..

Reply:
Alright, I'm fine with sticking to it for consistency.